settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'java-ilp-core'
```

Comment:
I suspect we'll be moving these repos over to an InterledgerJava project so could wait until then.

Just seems weird to me to have a library in MavenCentral called "java*".

Reply:
> It feels a little weird to have the github project called something different than the maven coordinates, which are currently generated from this value.

The Github project is just a "directory" the code sits in. This is necessarily prefixed "java" because its in the Interledger group and there could be other folders in the group for other languages.

If we moved this to an all-Java group it would seem weird to have to rename the Maven coordinates and also weird to still call this java-ilp-core.

If this is going to break stuff then let's take it out but my thinking is we should change it sooner rather than later.

Reply:
Yep, makes sense. I'm good with the module rename.